tests/test_deprecations.py

```diff
@@ -246,3 +246,39 @@ def test_signal_len():
         with pytest.raises(TypeError, match=re.escape("had no len()")):
             # remove Signal.__len__ from pyfar 0.8.0!
             len(pf.Signal([1, 2, 3], 44100))
+
+
+def test_deprecations_find_nearest_k():
```

Comment:
Can you please add the test for checking if the attributes were deprecated everywhere. There are examples above starting with `if version.parse(pf.__version__) >= version.parse('0.8.0'):`. Otherwise we might forget to deprecate later...